src/collections/_documentation/workflow/alerts-notifications/index.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Alerts & Notifications
+sidebar_order: 0
+---
+
+{% capture __alert_content -%}
```

Comment:
this section will be replaced with the alert specific email, right?

Reply:
Yes. 😄